### ISSUE TITLE
Add ARIA labels and screen reader support to game cells (Accessibility (a11y) Improvements sub-issue)

### ISSUE THREAD

**Issue by Lanthoiba2022** [notakto/notakto-website#349]
Right now Game cells lack proper ARIA labels and descriptions making it difficult for screen readers.

This improvement will make Notakto playable for users with visual impairments and other disabilities, significantly expanding the game's accessibility and inclusivity.

**Comment by Lanthoiba2022** [comment]
@AshmitSherigar Hi would love to solve this issue.

**Comment by AshmitSherigar** [comment]
@Lanthoiba2022 you can start working on this !

**Comment by AshmitSherigar** [comment]
> I’m really interested in working on this issue I would really appreciate it if you could assign it to me. I’m excited to help improve the project and will make sure to follow all contribution guidelines.

This issue is already assigned .. you could work on other a11y prblems or other open issues

**Comment by Rakshitg600** [comment]
@Lanthoiba2022 update?

**Comment by Lanthoiba2022** [comment]
@Rakshitg600 Hi I have been occupied with some work so not much progress. Please let others take it up if any. Thank you

**Comment by magic-peach** [comment]
@Rakshitg600 i would like to work on this issue as a part of ECWOC'26,please assign it to me

**Comment by Rakshitg600** [comment]
> [@Rakshitg600](https://github.com/Rakshitg600) i would like to work on this issue as a part of ECWOC'26,please assign it to me

Please solve one at a time, you can take it once your current one is done.
